modules/statics/src/coins.ts

```diff
@@ -283,7 +289,7 @@ export const coins = CoinMap.fromCoins([
   erc20('hydro', 'Hydro', 18, '0xebbdf302c940c6bfd49c6b165f457fdb324649bc', UnderlyingAsset.HYDRO),
   erc20('i8', 'i8 Exchange Token', 18, '0xb61bb66bf9caba531e6bb2fb75dec389a1664dfd', UnderlyingAsset.I8),
   erc20('idex', 'IDEX', 18, '0xb705268213d593b8fd88d3fdeff93aff5cbdcfae', UnderlyingAsset.IDEX),
-  erc20('idrc', 'Idrc', 18, '0x569f2241551011d5710c40fc61e0b3906c780b2f', UnderlyingAsset.IDRC),
+  erc20('idrc', 'Rupiah Coin', 18, '0x569f2241551011d5710c40fc61e0b3906c780b2f', UnderlyingAsset.IDRC),
```

Comment:
Confirming this name change is intentional

Reply:
yes